emission/net/ext_service/habitica/register.py

```diff
@@ -0,0 +1,24 @@
+# Standard imports
+import json
+import requests
+import logging
+
+# Our imports
+from sdk import Habitica
+from emission.core.get_database import get_habitica_db
+
+key_file = open('conf/net/keys.json')
+key_data = json.load(key_file)
+url = key_data["habitica"]["url"]
+
+def habiticaRegister(username, email, password, our_uuid):
```

Comment:
@sunil07t and @juemura need to decide login flow. will the user specify a password or will we autogenerate on? Everybody hates coming up with new passwords every time and requiring a separate password means that the integration is not as seamless as we would like.

On the other hand, if we want to use the standard habitica, maybe we want to allow existing users to login to their existing account?